x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -387,12 +388,49 @@ const eventsDefaultDataStream = {
   namespace: 'default',
 };
 
+enum AlertTypes {
+  MALWARE = 'MALWARE',
+  MEMORY_SIGNATURE = 'MEMORY_SIGNATURE',
+  MEMORY_SHELLCODE = 'MEMORY_SHELLCODE',
+}
+
 const alertsDefaultDataStream = {
   type: 'logs',
   dataset: 'endpoint.alerts',
   namespace: 'default',
 };
 
+const alertsDefaultDll = [
```

Comment:
can we not use mutable data since that could impact all generated data from this array?
instead, convert it to a function that returns this data.

Reply:
good idea. will do